.travis.yml

```diff
@@ -29,8 +29,10 @@ node_js:
 
 ## Use this to prepare your build for testing
 ## e.g. copy database configurations, environment variables, etc.
-# before_script:
-#
+## http://stackoverflow.com/questions/12336566/travis-ci-with-jasmine-node
+before_script:
```

Comment:
I don't see why not! I think the issue is making sure the server can be accessed from the tests. :<